app/frontend/Settings/Mail/Account/IdentityBlock.svelte

```diff
@@ -34,31 +34,31 @@
       -->
 
       {#if showReplyTo}
-        <label for="replyto" class="reply-to">Reply-to address</label>
+        <label for="replyto" class="reply-to">{$t`Reply-to address`}</label>
         <input type="email" bind:value={identity.replyTo} name="replyto" class="reply-to" />
       {/if}
 
       {#if showOrganisation}
-        <label for="organisation">Company</label>
+        <label for="organisation">{$t`Company`}</label>
         <input type="text" bind:value={identity.organisation} name="organisation" />
       {/if}
     </grid>
 
     {#if showSignature}
       <vbox class="signature">
-        Signature
+        {$t`Signature`}
         <vbox class="signature-editor-box">
           <HTMLEditorToolbar {editor} />
           <HTMLEditor bind:html={identity.signatureHTML} bind:editor />
           {#if showSentBy}
             <hbox class="sentBy">
               <div>
                 <!--<Icon data={logo} size="20px" />-->
-                Sent by <a href={siteRoot} target="_blank" style="color: #20AE9E"><strong><em>{appName}</em></strong></a>
+                {$t`Sent by`} <a href={siteRoot} target="_blank" style="color: #20AE9E"><strong><em>{appName}</em></strong></a>
```

Comment:
Please put the appname including the `<a>` tag as 1 single placeholder.
Do not use the `<T>` here.

(We will need the same string when sending email.)

Reply:
How do I put the appname inside an `<a>` tag and inside `$t``string`` `?

Reply:
Try:
```suggestion
                {@html $t`Sent by ${'<a href={siteRoot} target="_blank" style="color: #20AE9E"><strong><em>{appName}</em></strong></a>'}`}
```

Reply:
(We'll append the same string to outgoing HTML emails as signature)